flink-batch-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/GenericRow.java

```diff
@@ -0,0 +1,123 @@
+package org.apache.flink.api.java.io;
+
+import java.beans.Transient;
+import java.util.Arrays;
+
+import org.apache.flink.api.java.tuple.Tuple;
+
+/**
+ * This class represent a row as a POJO
+ * 
+ * */
+public class GenericRow implements java.io.Serializable {
```

Comment:
Can you use the Row type of the `flink-table` instead? If yes, please add the dependency to the `pom.xml` of `flink-jdbc`.